lymph/edge.py

```diff
@@ -91,16 +91,19 @@ def set_micro_mod(self, new_micro_mod: float) -> None:
             raise ValueError("Microscopic spread modifier must be between 0 and 1!")
 
         self.micro_mod = new_micro_mod
-    
-    def set_spread_prob(self, new_spread_prob: float) -> None:
-        """Set the spread probability of the edge."""
-        new_spread_prob = float(new_spread_prob)
 
+
+    def get_spread_prob(self) -> float:
+        """Return the spread probability."""
+        return self._spread_prob

```

Comment:
Maybe add a safety check:
```python
if not hasattr(self, "_spread_prob"):
    self._spread_prob = 0.
```
or something like that to make sure we don't run into problems when someone tries to get the spread probs before setting them.